src/main/java/org/folio/auth/authtokenmodule/tokens/RefreshToken.java

```diff
@@ -18,6 +18,8 @@ public class RefreshToken extends Token {
    */
   public static final String TYPE = "refresh";
 
+  private static final long EXPIRATION_SECONDS = 604800;
```

Comment:
Can you add a comment like /** 604800 seconds = 7 days */

Reply:
Yep

Reply:
@julianladisch  I updated readme to refer to the new OpenApi yaml since before the only documentation  of the endpoints were in the Readme. Now we have the yaml. Clarified the yaml  to include the token expirations. Also noticed that the tokenResponse.json wasn't being referred to properly in the  yaml. Fixed that.
    
Also noticed that the tokenResponse.json didn't have the new expiration properties  in it. Fixed that as well.

Finally moved the `tokenResponse` method in `RouteApi` to be below where it is called in both places to make the code easier to follow.